docs/sign-verify-images-pipeline.md

```diff
@@ -0,0 +1,135 @@
+# Notation sign task for Azure DevOps pipeline
+
+This document walks you through how to create an Azure pipeline to achieve the following goals:
+
+1. Build a container image and push it to Azure Container Registry
+2. Install Notation CLI, sign the image with Notation and Notation AKV plugin. The generated signature will be automatically pushed to Azure Container Registry
+
+## Prerequisites
+
+- You have created a Key Vault in Azure Key Vault and created a self-signed signing key and certificate. You can follow this [doc](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-tutorial-sign-build-push#create-a-self-signed-certificate-azure-cli) to create self-signed key and certificate for testing purposes. If you have a CA issued certificate, see this doc for details.
+- You have created a registry in Azure Container Registry
+- You have an Azure DevOps repository or GitHub repository
+
+## Create Service Connection
+
+Notation sign and verify command need the credential to access the registry. User should use [Docker task](https://learn.microsoft.com/azure/devops/pipelines/tasks/reference/docker-v2) to login and the Notation task can get the credential from pipeline context.
+
+Create a [Docker Registry service connection](https://learn.microsoft.com/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#docker-registry-service-connection) in Azure DevOps pipeline to grant the access permission to your Azure Container Registry for the Notation tasks. 
+
+- Select the **Settings** button in the bottom-left corner.
```

Comment:
Tell users where to find the "Settings"

Reply:
See ref: https://learn.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#create-a-service-connection